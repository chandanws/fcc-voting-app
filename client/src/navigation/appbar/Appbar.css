.appbar {
  border-bottom: 1px solid black;
}

.appbar__button {
  margin: 2rem;
}
