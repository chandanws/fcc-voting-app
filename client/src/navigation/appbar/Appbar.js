import React from "react";
import { Button } from "../../button/ButtonComponent";
import { Link } from "react-router-dom";
import "./Appbar.css";

const Appbar = props => {
  return (
    <div className="appbar">
      <Button className="appbar__button">Menu</Button>
      {/* Later add component Link to button */}
      <Button className="appbar__button">Home</Button>
    </div>
  );
};

export default Appbar;
