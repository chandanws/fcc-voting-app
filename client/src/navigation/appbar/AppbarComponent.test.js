import React from "react";
import Appbar from "./Appbar";
import { shallow } from "enzyme";

describe("<Appbar />", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Appbar />);
  });

  it("Should have 1 div ", () => {
    expect(wrapper.find("div").length).toBe(1);
  });
});
