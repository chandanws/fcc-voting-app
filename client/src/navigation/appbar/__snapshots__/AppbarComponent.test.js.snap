// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Appbar /> should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Appbar />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Button
          className="appbar__button"
          component="button"
          modifiers={
                    Array [
                              "stroked",
                            ]
          }
          onClick={undefined}
>
          Menu
</Button>,
        <Button
          className="appbar__button"
          component={[Function]}
          modifiers={
                    Array [
                              "stroked",
                            ]
          }
          to="/"
>
          Home
</Button>,
      ],
      "className": "appbar",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": "Menu",
          "className": "appbar__button",
          "component": "button",
          "modifiers": Array [
            "stroked",
          ],
          "onClick": undefined,
        },
        "ref": null,
        "rendered": "Menu",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": "Home",
          "className": "appbar__button",
          "component": [Function],
          "modifiers": Array [
            "stroked",
          ],
          "to": "/",
        },
        "ref": null,
        "rendered": "Home",
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Button
            className="appbar__button"
            component="button"
            modifiers={
                        Array [
                                    "stroked",
                                  ]
            }
            onClick={undefined}
>
            Menu
</Button>,
          <Button
            className="appbar__button"
            component={[Function]}
            modifiers={
                        Array [
                                    "stroked",
                                  ]
            }
            to="/"
>
            Home
</Button>,
        ],
        "className": "appbar",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": "Menu",
            "className": "appbar__button",
            "component": "button",
            "modifiers": Array [
              "stroked",
            ],
            "onClick": undefined,
          },
          "ref": null,
          "rendered": "Menu",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": "Home",
            "className": "appbar__button",
            "component": [Function],
            "modifiers": Array [
              "stroked",
            ],
            "to": "/",
          },
          "ref": null,
          "rendered": "Home",
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
